ipatests/test_integration/test_commands.py

```diff
@@ -577,3 +579,64 @@ def test_config_show_configured_services(self):
             # reset
             entry['ipaConfigString'] = orig_cfg
             conn.update_entry(entry)  # pylint: disable=no-member
+
+    def test_ssh_from_controller(self):
+        """https://pagure.io/SSSD/sssd/issue/3979
+        Test ssh from test controller after adding
+        ldap_deref_threshold=0 to sssd.conf on master
+
+        Steps:
+        1. setup a master
+        2. add ldap_deref_threshold=0 to sssd.conf on master
+        3. add an ipa user
+        4. ssh from controller to master using the user created in step 3
+        """
+        sssd_version = ''
+        cmd_output = self.master.run_command(['rpm', '-qa', 'sssd'])
```

Comment:
On fedora this command does not return anything, but on rhel it works. It seems safer to call `sssd --version` which works on all OSes (and return a string that can be parsed by parse_ipa_version).

Reply:
replaced `rpm -qa sssd` to `sssd --version`